swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2022-09-04/redhatopenshift.json

```diff
@@ -769,6 +806,10 @@
           "description": "The cluster properties.",
           "x-ms-client-flatten": true
         },
+        "installVersion": {
+          "description": "The cluster install version.",
+          "type": "string"
```

Comment:
again to the previous point, we should be clear whether the expected response is an array or a string

Reply:
(and I know this is auto-generated, but is is generating with the value we expect right now?)

Reply:
This is an OpenShift Clusters field. A field which specify the version of that particular OpenShift cluster .. which has to be a `string`.
https://github.com/anshulvermapatel/ARO-RP/blob/list-version-endpoint/pkg/api/v20220904/openshiftcluster.go#L18-L42
Here it is not intended to be a list of versions.